Misc/NEWS.d/next/Core_and_Builtins/2025-11-15-16-30-46.gh-issue-141594.PSsC5J.rst

```diff
@@ -0,0 +1 @@
+Add free-threading support to the JIT. Patch by Ken Jin.
```

Comment:
I think that you should clarify that this is only works for single threaded code.